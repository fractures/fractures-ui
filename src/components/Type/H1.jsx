import PropTypes from "prop-types"
import styled from "styled-components"

const propTypes = {
	children: PropTypes.node.isRequired,
	className: PropTypes.string
}

const H1 = styled.h1`
	margin-bottom: 0;
	margin-top: 0;

	font-size: 2.25rem;
	line-height: 2.8125rem;
`

H1.propTypes = propTypes

export default H1
