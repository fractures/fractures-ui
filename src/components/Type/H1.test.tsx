import H1 from './H1'

describe('H1', () => {
	const tree = renderer.create(<H1>Extra header</H1>).toJSON()

	it('Matches snapshot', () => {
		expect(tree).toMatchSnapshot()
	})
})
