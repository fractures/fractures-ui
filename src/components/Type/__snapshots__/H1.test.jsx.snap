// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H1 Matches snapshot 1`] = `
<h1
  className="sc-bdVaJa cCpoeI"
>
  Extra header
</h1>
`;
